config/manager-base/manager.yaml

```diff
@@ -48,7 +48,7 @@ spec:
               fieldRef:
                 fieldPath: metadata.namespace
           - name: RELATED_IMAGE_BUILD
-            value: gcr.io/kaniko-project/executor:latest
+            value: quay.io/buildah/stable:latest
```

Comment:
Is this the official image that is being used by the u/s Shipwright project?
(I don't remember if when we tested it it was with the u/s version of Shipwright)

Reply:
Ack. Then let's use the official one as you did. Is there a way to use a specific version instead of `latest`? I think we should use `v1.41.3` instead of `latest` - we don't want things to start breaking under our legs.